src/trackedit/internal/au3/au3trackeditclipboard.cpp

```diff
@@ -90,3 +75,68 @@ bool Au3TrackeditClipboard::isMultiSelectionCopy() const
 {
     return m_isMultiSelectionCopy;
 }
+
+std::set<int64_t> Au3TrackeditClipboard::getGroupIDs(std::vector<TrackData>& tracksData)
+{
+    std::set<int64_t> groupIds;
+
+    for (auto& data : tracksData) {
```

Comment:
data can be marked const as well
```suggestion
    for (const auto& data : tracksData) {
```